Sources/Code/Coordinator.swift

```diff
@@ -125,7 +125,8 @@ open class Coordinator {
     ///
     /// - Parameters:
     ///   - alreadyDisappeared: Dismisses or pops the view controller if set to `false`.
-    public func finish(alreadyDisappeared: Bool = false) {
+    ///   - animated: Animates the disabling of viewController
```

Comment:
Minor things, but could you please add a `.` at the end of the sentence? Also I think "disabling" isn't the best word to describe, what about "disappearance"? And "viewController" should be "the view controller" since this is plain English, no code.